roles/acm_hypershift/tasks/download-cli.yml

```
@@ -1,7 +1,10 @@
 ---
 - name: Download stable openshift client
   vars:
-    ocp_client_url: "https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable-4.14/openshift-client-linux.tar.gz"
+    ocp_client_url: >-
+      {{ "https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/" +
+      "stable/openshift-client-linux-amd64-rhel" +
+      ansible_distribution_major_version + ".tar.gz" }}
```

I think it only needed to remove the newline and remove the slash before the version i.e.
```suggestion
    ocp_client_url: >-
      https://mirror.openshift.com/pub/openshift-v4/
      x86_64/clients/ocp/stable/openshift-client-linux-amd64-rhel
      {{ ansible_distribution_major_version }}.tar.gz
```